src/components/TodoInput.js

```diff
@@ -0,0 +1,31 @@
+import { useContext, useState } from 'react';
+import { HiPlusCircle } from 'react-icons/hi';
+import TodoContext from '../TodoContext';
+
+const TodoInput = () => {
+  const [input, setInput] = useState('');
+  const { addTodo } = useContext(TodoContext);
+
+  const handleSubmit = () => {
```

Comment:
As the event you are handling is a click, the event handler's name should reflect that.